gapstool.vbs

```diff
@@ -10,17 +10,24 @@
 ' distributed under the License is distributed on an "AS IS" BASIS,
 ' WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 ' See the License for the specific language governing permissions and
-' limitations under the License.

+' limitations under the License.
+
 ' GAPS diagnostics tool
 ' Liron Newman lironn@google.com
 
 ' Do not change this line's format, build.bat relies on it.
-Const Ver = "1.0.1.0"

+Const Ver = "1.0.1.0"

```

Comment:
I have no idea. The only thing I can think of is that the line numbers changed slightly due to CR/LF issues. I tried saving my changes using Visual Studio Code, SublimeText, and Notepad, but they Git always showed these lines as being altered.

I'll take another look this evening.